example/src/main/java/co/infinum/example/SharedPrefs.java

```diff
@@ -81,4 +84,36 @@ public static void setPin(String pin) {
     public static void setRxExample(boolean isRxExample) {
         PREFS.edit().putBoolean("isRx", isRxExample).commit();
     }
+
+    public static void setStrongAuth(boolean strongAuth) {
+        PREFS.edit().putBoolean("strongAuth", strongAuth).commit();
+    }
+
+    public static boolean getStrongAuth() {
+        return PREFS.getBoolean("strongAuth", false);
```

Comment:
Can we make some private constants here instead of hardcoded keys?

Reply:
Sure, I was just going with the flow here since it is a sample app it was not so important to me to fix it, but why not spend some little effort to improve it!